client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -432,8 +433,10 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                 />;
             case TableHeadersNames.multipleCheck:
                 const isGroupShown = checkGroupedInvestigationOpen.includes(indexedRow.epidemiologyNumber);
+                const { isSelfInvestigated , selfInvestigationStatus ,selfInvestigationUpdateTime } = indexedRow;
                 return (
-                    <>
+                    <Box flexWrap="nowrap" display="flex" alignItems="center">
```

Comment:
Move the SelfInvestigationIcon to the left of the checkbox. 
It will look better if all the checkboxes will be aligned.

Reply:
added [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1093/commits/d69e1eb61c0dbd9dddd5c35c1cb3dd6a01efcfa2), looks much nicer now